app/(main)/(routes)/app/_components/Send.tsx

```diff
@@ -18,8 +18,9 @@ const Send = () => {
   const [emailSent, setEmailSent] = useState<boolean>(false);
   const [countdown, setCountdown] = useState<number | null>(null);
   const [amount, setAmount] = useState<number | undefined>(undefined);
-  const [selectedAssetString, setSelectedAssetString] =
-    useState<string>("TEST"); // Selected ERC20
+  const [selectedAssetString, setSelectedAssetString] = useState<
+    string | null
+  >(); // Selected ERC20
```

Comment:
This fails ESlint:
```
./app/(main)/(routes)/app/_components/Send.tsx:333:27
Type error: 'selectedAssetString' is possibly 'null' or 'undefined'.
  331 |                     const assetID =
  332 |                       assetType === "ERC20"
> 333 |                         ? selectedAssetString.toString()
      |                           ^
  334 |                         : selectedNFT?.contractAddress || ""; // Using enum name instead of index
  335 |                     const recipientAddr = recipient; // Since we're sending to an email, recipient address is the email itself
  336 |                     const sendFunction =
error Command failed with exit code 1.
```

Reply:
You also aren't following style convention on this line compared to the rest of the code (i.e. 3 lines instead of 1). Why did you choose to do that? Can you add a linter pre-commit hook to ensure you don't do the same in the future?

Reply:
Do you have eslinter enabled? if not, can you put that in the vscode config and the precommit hook so that this doesn't happen again?

Reply:
Bump @wryonik